languages/en-US/moderation/kick.json

```
@@ -1,10 +1,10 @@
 {
     "DESCRIPTION": "Kick the mentioned member!",
-    "USAGE": "{{prefix}}kick [@user] (reason)",
+    "USAGE": "{{prefix}}kick [@member] (reason)",
     "EXAMPLES": "{{prefix}kick @Androz#2091 Spam",
     "MISSING_MEMBER": "Please specify a valid member to kick!",
-    "MISSING_PERM": "An error has occurred... check that I have the permissions to kick this member and try again!",
-    "KICKED_DM": "Hello {{username}},\nYou have just been kicked from **{{server}}** by **{{moderator}}** for **{{reason}}**!",
-    "KICKED": "**{{username}}** has just been kicked from **{{server}}** by **{{moderator}}** for **{{reason}}**!",
+    "MISSING_PERM": "An error has occurred... Please check that I have the `Kick Members` permission and try again!",
```

Le bot a forcément la permission `KICK_MEMBERS`, mais pas forcément les permissions d'expulser ce membre en particulier
```suggestion
    "MISSING_PERM": "An error has occurred... check that I have the permissions to kick this member and try again!",
```

---

Autant pour moi je ne savais pas.

---

b535d6f82539c53002e1613fcd3690af22861f41